metadata_service/proxy/neo4j_proxy.py

```diff
@@ -827,7 +836,8 @@ def _build_user_from_record(record: dict, manager_name: str = '') -> UserEntity:
                           slack_id=record.get('slack_id'),
                           employee_type=record.get('employee_type'),
                           role_name=record.get('role_name'),
-                          manager_fullname=record.get('manager_fullname', manager_name))
+                          manager_fullname=record.get('manager_fullname', manager_name),
+                          other_key_values=other_key_values)
```

Comment:
maybe put up / update the doc for user model on how we could add new attribute without updating the model

Reply:
Sounds good. Will add.